nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -293,14 +264,7 @@ define([], () => {
                 },
                 {
                     mode: 'error',
-                    stage: 'queued',
-                },
-                {
-                    mode: 'error',
-                    stage: 'running',
-                },
-                {
-                    mode: 'error',
```

Comment:
Combined two error states into a single one -- `mode: 'error', stage: 'queued'` and `mode: 'error', staged: 'running'` ==> `mode: 'error', stage: 'runtime'`. `mode: 'error'` was never used, so deleted it.